sbin/Makefile.am

```diff
@@ -55,10 +52,65 @@ if ENABLE_VNETBUILD
 scripts += vnetbuild
 endif
 
+
+if FILENAME_VERSIONS
+
+install-exec-hook:
+if ENABLE_FIREHOL
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f firehol firehol-@PACKAGE_VERSION@ && \
+		$(LN_S) firehol-@PACKAGE_VERSION@ firehol
+endif
+if ENABLE_FIREQOS
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f fireqos fireqos-@PACKAGE_VERSION@ && \
+		$(LN_S) fireqos-@PACKAGE_VERSION@ fireqos
+endif
+if ENABLE_LINK_BALANCER
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f link-balancer link-balancer-@PACKAGE_VERSION@ && \
+		$(LN_S) link-balancer-@PACKAGE_VERSION@ link-balancer
+endif
+if ENABLE_UPDATE_IPSETS
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f update-ipsets update-ipsets-@PACKAGE_VERSION@ && \
+		$(LN_S) update-ipsets-@PACKAGE_VERSION@ update-ipsets
+endif
+if ENABLE_VNETBUILD
+	cd $(DESTDIR)$(sbindir) && \
+		mv -f vnetbuild vnetbuild-@PACKAGE_VERSION@ && \
+		$(LN_S) vnetbuild-@PACKAGE_VERSION@ vnetbuild
+endif
```

Comment:
I do not quite understand this.

I thought that we install package into /usr/libexec/firehol/firehold-@PACKAGE_VERSION@/* always, while symlink programs into /sbin.

If we generate the templates in one file, for example firehol.config, we can do $(dirname "$(readline "$0")") and to obtain firehol.config which will contain only autoconf variables, then there is no need to check for the AUTOCONF_RUN, all scripts can be pure scripts. if the firehol.config is missing then the autoconf was not run. this firehol.config is the one that its location can be overridden by environment.

the eselect mechanism (if required) only manages the symlinks at the /sbin, the rename of the /sbin stuff should be left to packager.